application/src/main/resources/verification.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<h1> Welcome to Pilot! </h1>
+<p> Click the below link to verify your account. </p>
+<a href="http://thunder.sanctionco.com/verify?email=%s&token=%s&response_type=html">Click here to verify your account!</a>
```

Comment:
We need to figure out what to do about this. Maybe put a placeholder (something like `URL`) here in the file, and then in the code do a `string.replace("URL", "correct-url-here")`?

Reply:
Yea, that may need to happen for now.

Reply:
Will you address this now or in another PR? If in another, can you open an issue once this is merged?